spec/system/jobseekers/jobseekers_can_manage_a_profile_spec.rb

```diff
@@ -13,17 +14,6 @@
 
     after { logout }
 
-    it "allows the jobseeker to navigate to their profile" do
```

Comment:
Added this to an existing test, I don't think it needed it's own description/set up